index.html

```diff
@@ -0,0 +1,124 @@
+            <li class="optionsheader2">Sponsor</li>
+            <li class="optionsheader2">News</li>
+            <li class="campaign">CC Campaign</li>
+          </ul>
+        </div>
+
+        <div class="header3">
+          <p class="hello">"Hello! Sharing a world concert in Bolivia"</p>
+          <p class="massive">WELCOME TO THE MASSIVE LIVE CONCERT VERSION<br>2022</p>
+          <p class="extreme">An extreme massive live concert gathering world class artists for the rejoice of those people who loves good music and good performances transmitted to a more than 80 countries all around the world. </P>
+          <p class="date">2022.10.14(THU) ~ 15(FRI)</p>
+          <p>Estadium Hernando Siles, La Paz - Bolivia</p>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <section class="mainprogram">
+    <h1 id="mainprog" class="mainprog">Performing Acts</h1>
+    <div class="maincontain">
+      <ul class="mainreferences">
+        <li><img src="./images/opening.png" alt="Billie Eilish"></li>
+        <li class="redtext">Billie Eilish</li>
+        <li class="normaltext">An American singer and songwriter. She is best known for her successful debut single, Ocean Eyes. She belongs to a family of musicians, she was destined to make a career in performing arts.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/secondact.png" alt="Lecture"></li>
+        <li class="redtext">U2</li>
+        <li class="normaltext">U2, Irish postpunk band that by the end of the 1980s had established itself not only as one of the world’s most popular bands but also as one of its most innovative</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/thirdact.png" alt="Lecture"></li>
+        <li class="redtext">Madonna</li>
+        <li class="normaltext">Madonna, American singer, songwriter, actress, and entrepreneur immensely popular in the 1980s and ’90s becoming to be the "queen of pop" until our days.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/fourthact.png" alt="Lecture"></li>
+        <li class="redtext">Coldplay</li>
+        <li class="normaltext">Coldplay, British rock group whose melodic piano-driven anthems lifted it to the top of the pop music world in the early 21st century, formed in 1998, and leaded by pianist-vocalist Chris Martin.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/fifthact.png" alt="Lecture"></li>
+        <li class="redtext">Mariah Carey</li>
+        <li class="normaltext">Mariah Carey, American pop singer, noted for her remarkable vocal range. She was one of the most successful female performers of the 1990s and remained popular into the early 21st century.</li>
+      </ul>
+    </div>
+    <p class="tickets">Where to Buy Tickets</p>
+  </section>
+
+  <section class="moreinfo">
+    <h1 class="moretitle">More of our Performing Acts</h1>
+    <div class="maincards">
```

Comment:
- [ ] Kindly try to decrease the size of the image and add the background image to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/169205258-600e6fd5-3a0b-48c0-9c85-d952f6c7fa05.png) | ![image](https://user-images.githubusercontent.com/24703171/169205392-b5ec94fb-3a11-4f12-9d03-aa1e76c5f548.png) |